sdks/python/apache_beam/runners/worker/data_plane.py

```diff
@@ -639,8 +639,13 @@ def _write_outputs(self):
       streams = [self._to_send.get()]
       try:
         # Coalesce up to 100 other items.
+        total_size_bytes = 0
         for _ in range(100):
-          streams.append(self._to_send.get_nowait())
+          data_or_timer = self._to_send.get_nowait()
+          total_size_bytes += data_or_timer.ByteSize()
+          streams.append(data_or_timer)
+          if total_size_bytes > _DEFAULT_SIZE_FLUSH_THRESHOLD:
```

Comment:
could also use a larger value here.

-  2*_DEFAULT_SIZE_FLUSH_THRESHOLD (since we collect Data and Timers but then separate them in different streams)
- XXX MB

Reply:
_DEFAULT_SIZE_FLUSH_THRESHOLD is fine. This was mostly to colocate lots of tiny results for streaming pipelines (though that might be less valuable now that we can inline them into the control messages).